src/GitLink.Tests/Providers/CustomRawUrlProviderFacts.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace GitLink.Tests.Providers
+{
+    using GitLink.Providers;
+    using NUnit.Framework;
+
+    public class CustomRawUrlProviderFacts
+    {
+        [TestFixture]
+        public class TheInitialization
+        {
+            [TestCase]
```

Comment:
That was a mistake made by us. Please make the change, I will change the other tests after that.